src/components/admin/rich-text-input/RichTextInput.test.tsx

```
@@ -0,0 +1,407 @@
+            <button disabled={disabled} aria-label="Italic" title="Italic (Ctrl+I)">
+                <em>I</em>
+            </button>
+            <button disabled={disabled} aria-label="Line break" title="Line break">
+                ↵
+            </button>
+        </div>
+    );
+    const MockInitialValuePlugin = (props: any) => {
+        mockInitialValuePluginProps = props;
+        // Render a hidden element with the value for testing purposes
+        return props.value ? <div data-testid="initial-value-content" dangerouslySetInnerHTML={{ __html: props.value }} /> : null;
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)